libs/domains/environment/src/lib/slices/environments.slice.ts

```diff
@@ -18,8 +18,9 @@ export const environmentsAdapter = createEntityAdapter<EnvironmentEntity>()
 
 export const fetchEnvironments = createAsyncThunk<Environment[], { projectId: string }>(
   'environments/fetch',
-  async (data) => {
+  async (data, thunkApi) => {
     const response = await environmentsApi.listEnvironment(data.projectId).then((response) => response.data)
+    thunkApi.dispatch(fetchEnvironmentsStatus({ projectId: data.projectId }))
```

Comment:
Why when we fetch environments you fetch obligatory the status too?

I don't know if it's a good practice, to the example in the breadcrumb, for now, we didn't need the status of the environment. I think it's better if the environments fetch work alone or we can maybe create a new `createAsyncThunk` with these two requests

Reply:
> And, we agree with this improvement you are ok with rendering the table row with one skeleton?

Hello Remi, well, no. For me it's two different things and I still think that we must print the table as soon as the elements are available. If the status takes a lot of time to answer when the applications or environments are ready, we should print it and puting a skeleton or a spinner or I don't know what in the table. We will discuss this point with the rest of the team and we will see what we decide together

Reply:
It's part of my assumption. I thought that for the status it makes sense to always fetch them because anyway we are always needed the status, even if they're not needed in the breadcrumb, they are instantly needed when we load the view after a click on the breadcrumb.
But I understand that it's maybe not good.
I would like to avoid to duplicate the ayncthunk though, it makes us duplicate the logic in the fulfill for not too much reasons... I don't know what to do here